### ISSUE TITLE
Screen readers can read content outside the "add user" modal

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4102]
## Area or Component
Manage users page - "add user" modal (but probably all modals on the page)

## Quick Summary
Screen readers can read content outside the modal for "add users".

## Accessibility Violation
Able to browse outside modal with screen reader: https://cdc-devtools.dequecloud.com/rule-help/en/reading-order-browse-outside-modal

## Action Requested
All other entries on the page should be `aria-hidden:true`. See #4075 for more information (and hopefully a solution?)

This needs to be fixed for all modals on the manage users page.

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=1647999156&range=15:15

**Comment by georgehager** [comment]
Based on research done by @mehansen, we think this violation is a false positive. I've confirmed that:
- The "Add user" model does use the accessible `react-modal` component 
- When testing with a screenreader, I am not able to read outside of the "Add user" modal

Based on this, I'm putting this on hold until we hear back from Deque. See more context in #4053.

cc: @emmastephenson

**Comment by georgehager** [comment]
Further evidence that this is likely a non-issue from running AXE DevTools locally (none of the elements that they list for checking are tabbable):

![Screen Shot 2022-09-21 at 3 33 05 PM](https://user-images.githubusercontent.com/89797785/191594351-732a756c-f500-445a-a4f0-de326716d638.png)

**Comment by georgehager** [comment]
Also pointing out via the documentation:

>It is important for users of screenreaders that other page content be hidden (via the aria-hidden attribute) while the modal is open. To allow react-modal to do this, you should call Modal.setAppElement with a query selector identifying the root of your app. For example, if your app content is located inside an element with the ID root, you could place the following call somewhere in your code before any modals are opened:
>
> `Modal.setAppElement('#root');`

We currently have `Modal.setAppElement('#root');` set in our [frontend/src/index.tsx](https://github.com/CDCgov/prime-simplereport/blob/main/frontend/src/index.tsx) file.

**Comment by georgehager** [comment]
Confirming this a false positive corroborated by #4053, and closing!

cc: @emmastephenson

**Comment by georgehager** [comment]
To do due diligence, we followed up on this ticket by testing with NVDA. We were not able to read outside the modal when using NVDA, see the screen recording for the Add User modal [here](https://drive.google.com/file/d/10P_KnptfGFIFEN5MJLmexlA_nzy9gkSe/view?usp=sharing)
